src/app/project/project-details-tab/project-details-tab.component.ts

```diff
@@ -156,7 +163,7 @@ export class ProjectDetailsTabComponent implements OnInit, AfterViewInit, OnDest
     if (this.shapefile[0] && this.shapefile[0].documentFileName && this.shapefile[0].documentFileName.length > 0) {
```

Comment:
again might want to consider The optional chaining operator (?.)

Reply:
Not implementing for now. Would have to redo the entire conditional function based on this change, which I feel is out of scope for a bug fix.